internal/pebble/metrics.go

```diff
@@ -33,27 +35,28 @@ type metrics struct {
 	obsoleteWALCount   prometheus.Gauge
 }
 
-func newMetrics() (*prometheus.Registry, *metrics, error) {
-	r := prometheus.NewRegistry()
-	writeStall, err := metric.NewAverager(
-		"pebble_write_stall",
-		"time spent waiting for disk write",
-		r,
-	)
-	if err != nil {
-		return nil, nil, err
-	}
-	getLatency, err := metric.NewAverager(
-		"pebble_read_latency",
-		"time spent waiting for db get",
-		r,
-	)
-	if err != nil {
-		return nil, nil, err
-	}
+func newMetrics(r prometheus.Registerer) (*metrics, error) {
 	m := &metrics{
-		writeStall: writeStall,
-		getLatency: getLatency,
+		writeStallCount: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: "pebble",
```

Comment:
Wondering if passing in a namespace from the caller here makes more sense as well.

Reply:
This change was just intended to remove the multi gatherer usage, other than that it should be a no-op so I didn't add registration of any metrics that was previously ignored or change any namespaces.